### DISCUSSION TITLE
Interactive viewer based on vtk+vedo

### DISCUSSION THREAD

**Discussion by nantille** [marcomusy/vedo#424] [Show and tell]
Hey Marco,

I wanted to thank you for your efforts in making vedo and in solving the issues I reported. Thanks to your tool and VTK, I've succeeded in making an interactive viewer, plainly called [iblviewer](https://github.com/int-brain-lab/iblviewer).

The viewer is a general-purpose one with some part dedicated to neuroscience. Its architecture is specifically made as an interactive application rather than a one-time plotter. If a good part of the tool relies directly on VTK (especially for volume-related rendering and slicing), another part relies on vedo for its simplicity. There might be interesting parts that you could use for improving vedo. :)

Cheers

**Comment by marcomusy** [comment]
Thanks Nicolas! I'll have a look at it!
